### PR TITLE
Update epa_eia_crosswalk name

### PR DESCRIPTION
**Pull request by aesharpe**
I updated the name `epa_eia_crosswalk` in the pudl and archiver repo to be more specific. Now it's `epacems_unitid_eia_plant_crosswalk`. This PR changes any references to the old name.

### CODE REVIEW

File: pudl_scrapers/spiders/epa_eia_crosswalk.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 """
```
Comment (zaneselvans) [inline pudl_scrapers/spiders/epa_eia_crosswalk.py:1]: Need to change the name of the module to match the name of data source.

**Review by zaneselvans — CHANGES_REQUESTED** [review]
I think the only change required here is to update the module name to reflect the new data source name.

File: pudl_scrapers/spiders/epa_eia_crosswalk.py (hunk 1 above)
Comment (aesharpe) [reply pudl_scrapers/spiders/epa_eia_crosswalk.py:1]: Ahh good catch

**Review by zaneselvans — APPROVED** [review]

File: pudl_scrapers/items.py [hunk 2]
```diff
@@ -69,34 +69,13 @@ def __repr__(self):
...
 
 
-class EipInfrastructure(DataFile):
-    """The EipInfrastructure excel file."""
-
-    def __repr__(self):
-        return "EipInfrastructure('%s')" % self["save_path"]
-
-
 class LblnIsoQueues(DataFile):
```
Comment (zaneselvans) [inline pudl_scrapers/items.py]: Should this stick around? If so, it should be LBNL (Lawrence Berkeley National Labs) not LBLN

**Review by zaneselvans — APPROVED** [review]
It seemed like one LBNL (but misspelled LBLN) entity was sticking around. Should it be removed? In `pudl_scrapers/items.py`

File: pudl_scrapers/items.py (hunk 2 above)
Comment (aesharpe) [reply pudl_scrapers/items.py]: good catch -- it should go

**Pull request merged**